src/Middleware/RetryMiddleware.php

```diff
@@ -51,14 +51,23 @@ class RetryMiddleware
     /** @var float|null */
     private $deadlineMs;
 
+    /*
+     * The number of retries that have already been attempted.
+     * The first retry will have $retryAttempts set to 0.
+     * @var int
+     */
+    private int $retryAttempts;
```

Comment:
We don't have other precedents of adding type hints in php7.4 style in this repo. We can either choose to keep `private int` or the `@var int`, both seems redundant.

```suggestion
    private $retryAttempts;
```

Reply:
I disagree here. The reason the precedent is to not have typehints is because we previously were supporting PHP versions which didn't support typehints, so adding them would have thrown a PHP Syntax Error. However, we've now dropped support for those versions of PHP, and adding typehints is the _new_ precedent. I even have a PR which adds typehints to all properties (https://github.com/googleapis/gax-php/pull/452). 

So, when adding new properties, we absolutely want to add typehints.